arangod/Aql/OptimizerRules.h

```diff
@@ -238,6 +238,10 @@ void fulltextIndexRule(aql::Optimizer*, std::unique_ptr<aql::ExecutionPlan>, aql
 /// @brief push LIMIT into subqueries, and simplify them
 void optimizeSubqueriesRule(Optimizer*, std::unique_ptr<ExecutionPlan>, OptimizerRule const*);
 
+/// @brief replace legacy JS functions in the plan.
+void replaceJSFunctions(Optimizer*, std::unique_ptr<ExecutionPlan>, OptimizerRule const*);
```

Comment:
choose a better name. there really is no need to mention js here - its rather AQL functions you replace.